corehq/apps/userreports/README.rst

```diff
@@ -1837,6 +1837,51 @@ Rounds decimal and floating point numbers to two decimal places.
        "custom_type": "short_decimal_display"
    }
 
+
+Prefix a string
+~~~~~~~~~~~~~~~
+
+Prefixes the column value with another string
+
+.. code:: json
+	{
+        "type": "prefix_string",
+        "prefix": "hello "
+    }
+
+Convert into a link
+~~~~~~~~~~~~~~~~~~~
+
+Display a text as a hyperlink
+
+.. code:: json
+
+   {
+       "type": "hyperlink",
+       "link_text": "View Form",
+   }
```

Comment:
The elegance / purity of this is nice, but I think it'd be easier to use and read if it were more like:

```json
{
   "type": "hyperlink",
   "link_text": "View Form",
   "link_template": "https:///www.commcarehq.org/a/cory/forms/{value}/"
}
```
Or something like that. Any reason you went with this approach?